package.json

```diff
@@ -28,12 +28,11 @@
     "rebuild-hard:dev": "npm run clean-slate && npm run build:dev",
     "rebuild-hard:prod": "npm run clean-slate && npm run build:prod",
     "precommit": "lint-staged",
-    "prepush": "lint-staged",
-    "prettier:base": "prettier --parser typescript --single-quote --trailing-comma es5 --tab-width 2 --no-semi --print-width 80 --write",
```

Comment:
For some reason, when the `--parser` isn't explicitly set, prettifier doesn't work on my machine. I know it's supposed to infer it, but setting explicitly meant I didn't have to figure out why it didn't work.